src/app/hooks/useTransactions.ts

```diff
@@ -17,16 +18,38 @@ export const useTransactions = () => {
           accountId,
           limit,
         });
+        const fetchInvoice = await api.getInvoices({ isSettled: true, limit });
+
+        const invoices: Transaction[] = fetchInvoice.invoices.map(
+          (invoice) => ({
+            ...invoice,
+            title: invoice.memo,
```

Comment:
are both title and description supposed to be the same?

Reply:
we did that before as well. we only have the memo field in the invoice which is meaninfull so we set title and description both as memo